devtools/cli/src/main/java/io/quarkus/cli/plugin/CliPluginsList.java

```diff
@@ -0,0 +1,129 @@
+package io.quarkus.cli.plugin;
+
+import static io.quarkus.devtools.utils.Patterns.isExpression;
+import static io.quarkus.devtools.utils.Patterns.toRegex;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.TreeMap;
+import java.util.concurrent.Callable;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+import io.quarkus.cli.common.RunModeOption;
+import io.quarkus.devtools.project.BuildTool;
+import io.quarkus.runtime.util.StringUtil;
+import picocli.CommandLine;
+
+@CommandLine.Command(name = "list", aliases = "ls", header = "List CLI plugins. ")
+public class CliPluginsList extends CliPluginsBase implements Callable<Integer> {
+
+    @CommandLine.Mixin
+    RunModeOption runMode;
+
+    @CommandLine.Option(names = { "-i",
+            "--installable" }, defaultValue = "false", order = 4, description = "List plugins that can be installed")
+    boolean installable = false;
+
+    @CommandLine.Option(names = { "-s",
+            "--search" }, defaultValue = "*", order = 5, paramLabel = "PATTERN", description = "Search for matching plugins (simple glob using '*' and '?').")
+    String searchPattern;
+
+    @CommandLine.Option(names = { "-c",
+            "--show-command" }, defaultValue = "false", order = 6, description = "Show the command that corresponds to the plugin")
+    boolean showCommand = false;
+
+    Map<String, Plugin> installedPlugins = new HashMap<>();
+
+    @Override
+    public Integer call() {
+        try {
+            output.debug("List extensions with initial parameters: %s", this);
+            output.throwIfUnmatchedArguments(spec.commandLine());
+
+            if (runMode.isDryRun()) {
+                return dryRunList(spec.commandLine().getHelp(), null);
+            }
+            Integer exitCode = listPluigns();
+            printHints(!installable && installedPlugins.isEmpty(), installable);
```

Comment:
even if there are installed plugins it should print the hint on adding --installable IMO.

```
qss plug list
    Name 	 Type  	 Scope   	 Location               	 Description
 *  kill 	 jbang 	 project 	 quarkus-kill@quarkusio
 ```
 
 if you are new to project but someone already enabled plugins you don't get a hint for it